save-preprocessor/src/main/resources/TestSuitesRepos

```diff
@@ -1 +0,0 @@
-https://github.com/saveourtool/save-cli origin/main examples/kotlin-diktat;examples/discovery-test
```

Comment:
Why is this file deleted? I can see references to `TestSuiteRepos.json`, but no new file. What is a strategy to work with standard suites in phase 3? Or is it already described in the issue?

Reply:
standard test suites will be added to `CQFN.org` as public tests and will be available to select for all users
right now, I move all these constants to hardcoded values on backend side. can extract to a file, but it will be not applicable in phase 3.

Also, will describe steps for phase 3 in #250